pkg/apis/nmstate/v1alpha1/condition_types_test.go

```diff
@@ -0,0 +1,98 @@
+package v1alpha1
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Conditions", func() {
+	Context("when the list is empty", func() {
+		It("should return nil when finding a condition", func() {
+			condition := ConditionList{}.Find(NodeNetworkStateConditionFailing)
+			Expect(condition).To(BeNil())
+		})
+	})
+
+	Context("when composed by one condition with known type and unknown status", func() {
+		originalConditions := ConditionList{
```

Comment:
I don't want to use the tested method to prepare for the test. If the Set is faulty, it would be difficult to find it.

They are the same, both are at the beginning of the epoch. I don't see how cloning from one to another would be better.

Reply:
Ok, but call time.Unix once and set both time with it, to ensure they are the same.

Reply:
I explictly set time.Unix to `0`, they will both be the beginning of the epoch. I think it is more readable like this than to confuse people with pre-defining of the Time, clonning and only then assigning.